lib/molecules/menu/template.html

```diff
@@ -1,13 +1,13 @@
 <ul ng-if="lnItems.length">
   <li ng-repeat="item in lnItems" ng-class="item.class">
-    <a ng-href="{{item.link}}" ng-attr-target="{{item.new_window ? '_blank' : ''}}" >
+    <a ng-href="{{item.link}}" ng-class="{{lnParentClass}}" ng-attr-target="{{item.new_window ? '_blank' : ''}}" >
```

Comment:
The ng-class does not need the `{{...}}`. This should be just `ng-class="lnParentClass"`.